### PR TITLE
feat(console): organization details

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

- add `<OrganizationRolesSelect />` component
- init organization details page with settings and members tab
- allow user organization roles to be edited

large PR already. missing tests will be added later.

## Testing

- can view organization details
- can change organization name or description
- can edit organization roles in the organization members tab

image

image

image

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [x] necessary TSDoc comments

### CODE REVIEW

File: packages/console/src/pages/OrganizationDetails/Members/index.tsx
Comment (gao-sun) [inline packages/console/src/pages/OrganizationDetails/Members/index.tsx:150]: next PR

File: packages/console/src/pages/OrganizationDetails/index.tsx
Comment (charIeszhao) [inline packages/console/src/pages/OrganizationDetails/index.tsx:86]: It would be better to wait until #4712 is merged and rebase from that, in order to have better support for the responsiveness of the details page

File: packages/console/src/pages/OrganizationDetails/Members/EditOrganizationRolesModal.tsx
Comment (charIeszhao) [inline packages/console/src/pages/OrganizationDetails/Members/EditOrganizationRolesModal.tsx:66]: Use i18n?

File: packages/console/src/pages/OrganizationDetails/index.tsx
Comment (gao-sun) [reply packages/console/src/pages/OrganizationDetails/index.tsx:49]: forgot that

File: packages/console/src/pages/OrganizationDetails/Members/EditOrganizationRolesModal.tsx
Comment (gao-sun) [reply packages/console/src/pages/OrganizationDetails/Members/EditOrganizationRolesModal.tsx:66]: added to slack

File: packages/console/src/pages/OrganizationDetails/index.tsx
Comment (gao-sun) [reply packages/console/src/pages/OrganizationDetails/index.tsx:86]: if you the component has no breaking changes i think it doesn't matter?

Comment (charIeszhao) [reply packages/console/src/pages/OrganizationDetails/index.tsx:86]: Yeah, sure. Let's talk about it later if necessary

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**
